test/email-sender.test.js

```diff
@@ -12,7 +12,7 @@ test.skip('Tests if sendMessage resolves with expected info', async () => {
   const testRecipient = process.env.NODEMAILER_TESTRECIPIENT;
   const testReturnValue = emailsender.sendMessage(testRecipient, testSubjectMessage, testHTML);
   // Tests if the expected resolve is correct
-  expect(testReturnValue).resolves.toBe([testRecipient]);
+  await expect(testReturnValue).resolves.toBe([testRecipient]);
```

Comment:
Should this `await` not go with the call to `emailsender.sendMessage` on line 13 above?